docs(FAQ): add instructions for installing UEK R7 on OL 8.6 (#1076)
accuknox/KubeArmor: getting-started/FAQ.md

If Block policy is used and there are no supported enforcement mechanism on the platform then the policy enforcement wouldn't be observed. But we will still be able to see the observability data for the applied Block policy, which can help us in identifying any suspicious activity.
</details>

<details>

>   <summary><h4>How to install UEK7 on Oracle Linux 8.6</h4></summary>

Reviewer: ```suggestion
  <summary><h4>How to use KubeArmor on Oracle K8s engine?</h4></summary>
```

> The default kernel for Oracle Linux 8.6 (OL 8.6) is UEK R6 kernel-uek-5.4.17-2136.307.3. BPF-LSM will not be present there. KubeArmor support enforcement on OKE leveraging the BPF-LSM.

Reviewer: ```suggestion
KubeArmor supports enforcement on OKE leveraging the BPF-LSM. The default kernel for Oracle Linux 8.6 (OL 8.6) is UEK R6 kernel-uek-5.4.17-2136.307.3 which does not support BPF-LSM.
```
